Samples/Hast.Samples.Consumer/SampleRunners/ImageSharpSampleRunner.cs

```diff
@@ -22,41 +22,43 @@ internal class ImageSharpSampleRunner : ISampleRunner
         // In case you wish to test the sample with a larger file, the fpga.jpg file must be replaced. You can find
         // a 100 megapixel jpeg here: https://photographingspace.com/100-megapixel-moon/
 
+        using var image = await Image.LoadAsync("fpga.jpg");
+
         // Not accelerated by Hastlayer.
-        RunSoftwareBenchmarks();
+        RunSoftwareBenchmarks(image);
 
         // Accelerated by Hastlayer.
-        using var image = await Image.LoadAsync("fpga.jpg");
-        var width = image.Width;
-        var height = image.Height;
+        var newWidth = image.Width / 2;
+        var newHeight = image.Height / 2;
 
-        var sw = Stopwatch.StartNew();
-        var newImage = image.Clone(img => img.HastResize(
-            width / 2, height / 2, Environment.ProcessorCount, hastlayer, hardwareRepresentation, configuration));
-        sw.Stop();
+        var stopwatch = Stopwatch.StartNew();
+        var newImage = image.Clone(context => context.HastResize(
+            newWidth, newHeight, Environment.ProcessorCount, hastlayer, hardwareRepresentation, configuration));
+        stopwatch.Stop();
         await newImage.SaveAsync("FpgaResizedWithHastlayer.jpg");
-        Console.WriteLine(FormattableString.Invariant($"On FPGA it took {sw.ElapsedMilliseconds} ms"));
+        Console.WriteLine(FormattableString.Invariant($"On FPGA it took {stopwatch.ElapsedMilliseconds} ms"));
     }
 
-    private static void RunSoftwareBenchmarks()
+    private static void RunSoftwareBenchmarks(Image image)
     {
-        using var image = Image.Load("fpga.jpg");
-        var width = image.Width;
-        var height = image.Height;
-
-        var sw = Stopwatch.StartNew();
-        var newImage = image.Clone(img =>
-            img.HastResize(width / 2, height / 2, Environment.ProcessorCount));
-        sw.Stop();
+        if (image == null) throw new ArgumentNullException(nameof(image));
```

Comment:
There is a built-in method for that.
```suggestion
        ArgumentNullException.ThrowIfNull(image);
```